app/src/main/java/com/github/epfl/meili/profile/friends/FriendsListActivity.kt

```diff
@@ -0,0 +1,71 @@
+package com.github.epfl.meili.profile.friends
+
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.github.epfl.meili.BuildConfig
+import com.github.epfl.meili.R
+import com.github.epfl.meili.database.FirestoreDatabase
+import com.github.epfl.meili.home.Auth
+import com.github.epfl.meili.models.Friend
+import com.github.epfl.meili.nearby.NearbyActivity
+import com.github.epfl.meili.util.MeiliViewModel
+import com.github.epfl.meili.util.TopSpacingItemDecoration
+
+class FriendsListActivity : AppCompatActivity() {
+    companion object {
+        private const val FRIENDS_PADDING: Int = 15
+    }
```

Comment:
Optional: you can add
```suggestion
    companion object {
        private const val FRIENDS_PADDING: Int = 15
        private const val TAG: String = "FriendListActivity"
        private const val TITLE: String = "My Friends"     
    }
```
And change the rest of the class accordingly.